spec/models/location_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.describe Location, type: :model do
+  it 'is valid with valid coordinates' do
+    expect(Location.new(latitude: 90, longitude: 180, locatable: TransportationOffice.new(name: 'Office'))).to be_valid
+    expect(Location.new(latitude: -90, longitude: -180, locatable: TransportationOffice.new(name: 'Office'))).to be_valid
+    expect(Location.new(latitude: 0, longitude: 0, locatable: TransportationOffice.new(name: 'Office'))).to be_valid
+  end
+
+  it 'is not valid with an invalid latitude' do
+    expect(Location.new(latitude: 91, longitude: 180, locatable: TransportationOffice.new(name: 'Office'))).to_not be_valid
+    expect(Location.new(latitude: -91, longitude: 180, locatable: TransportationOffice.new(name: 'Office'))).to_not be_valid
+  end
+
+  it 'is not valid with an invalid longitude' do
+    expect(Location.new(latitude: 90, longitude: 181, locatable: TransportationOffice.new(name: 'Office'))).to_not be_valid
+    expect(Location.new(latitude: 90, longitude: -181, locatable: TransportationOffice.new(name: 'Office'))).to_not be_valid
+  end
+end
```

Comment:
I believe you could DRY up some of this by stashing the `TransportationOffice` instance into a variable:

```rb
RSpec.describe Location, type: :model do
  let(:transportation_office) { TransportationOffice.new(name: 'Office') }

  it 'is valid with valid coordinates' do
    expect(Location.new(latitude: 90, longitude: 180, locatable: transportation_office)).to be_valid
    # etc. etc. etc.
  end

  # etc. etc. etc.
end
```